pmaweb/templates/contractor.html

```diff
@@ -0,0 +1,57 @@
+{% extends 'base.html' %}
+{% load staticfiles %}
+
+{% block content %}
+
+<h2>Hiring a contract developer</h2>
+
+<p>
+The phpMyAdmin project is often looking for developers who are interested in becoming contractors
+performing work on the project. Applicants who do not fit our needs at the time of application
+will be kept on file for a period of one year in the event the requirements change or a new position
+is available. Within a few days of applying, applicants can expect a response confirming that their
+submission was received.
+</p>
+
+<h3>Overview</h3>
+
+<p>
+The phpMyAdmin Project is looking for a full-time or part-time developer to assist in development, including bug fixing and refactoring.
+</p>
+
+<h3>Work plan</h3>
+
+<p>
+The ideal candidate will dynamically balance their workload based on outstanding issues and priority, but is
+anticipated to break down to these percentages (in priority order of what's most important to least important):
+</p>
+
+<ul>
+  <li>Security maintenance (5%)</li>
+  <li>Bug fixing and issue assessment (45%)</li>
+  <li>Code base improvement like refactoring and writing unit tests (45%)</li>
+  <li>Implementation of new features (5%)</li>
+</ul>
+
+<h3>Work conditions</h3>
+<ul>
+  <li>The Developer shall publish a weekly blog post about his/her work to the phpMyAdmin community.</li>
+  <li>The Developer shall send a monthly invoice to the Software Freedom Conservancy for the work done during that month.</li>
+  <li>All work produced by the Developer is to be licensed under "GPLv2 or later".</li>
+  <li>This is an independent contractor position: the Developer will be responsible for his/her own equipment and expenses.</li>
+</ul>
+
+<h3>The Candidate</h3>
+
+<p>
+The candidate will be able to demonstrate a very good knowledge of phpMyAdmin's code base. In addition, we expect
+excellent skills in all of the technologies used by phpMyAdmin (PHP, HTML, JavaScript, jQuery, CSS, MySQL) and excellent communication skills.
+</p>
+
+<h3>Applying</h3>
+
+<p>
+Candidates should submit their proposal (including their CV, availability, and financial terms) to: contract@phpmyadmin.net.
```

Comment:
Shouldn't there be "proposals"? (I'm not really sure, but I'd write it that way, but you're usually better in English than me ;-)).